src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/Completion/DirectiveAttributeCompletionItemProviderBaseTest.cs

```diff
@@ -16,108 +16,6 @@ public class DirectiveAttributeCompletionItemProviderBaseTest : RazorIntegration
 
         internal override bool UseTwoPhaseCompilation => true;
 
-        [Fact]
-        public void IntersectsWithAttributeNameOrPrefix_AtPrefixLeadingEdge_ReturnsFalse()
```

Comment:
Moved to `DirectiveAttributeCompletionTransition` as that's the only consumer of this function.